WebHandler/__init__.py

```diff
@@ -383,6 +406,177 @@ def get_highcourt_cases_by_name_wrapper():
                 {"status": True, "debugMessage": "Request Received and processing", "request": {"body": req_body, "params": req_params, "version": version}}),
         )
 
+    if req_params.get("method") == "districtcourt_advocatecasesbyname":

```

Comment:
this should not be a separate method from query params,
Client from query params method will be same as highcourts (advocatecasesbyname)
Internally we need to handle it.